src/components/Uploader/ImageQualityGuide.js

```diff
@@ -0,0 +1,164 @@
+              <div
+                role="img"
+                aria-label={imgAltText}
+                className={classNames(
+                  style.documentExampleImg,
+                  style.documentExampleImgCutoff
+                )}
+              />
+              <div
+                className={style.documentExampleLabel}
+                data-onfido-qa="documentExampleLabelCutoff">
+                {translate('image_quality_guide.not_cut_off')}
+              </div>
+            </div>
+            <div className={style.documentExampleCol}>
+              <div
+                role="img"
+                aria-label={imgAltText}
+                className={classNames(
+                  style.documentExampleImg,
+                  style.documentExampleImgBlur
+                )}
+              />
+              <div
+                className={style.documentExampleLabel}
+                data-onfido-qa="documentExampleLabelBlur"
+              >
+                {translate('image_quality_guide.no_blur')}
+              </div>
+            </div>
+          </div>
+          <div className={style.imageQualityGuideRow}>
+            <div className={style.documentExampleCol}>
+              <div
+                role="img"
+                aria-label={imgAltText}
+                className={classNames(
+                  style.documentExampleImg,
+                  style.documentExampleImgGlare
+                )}
+              />
+              <div
+                className={style.documentExampleLabel}
+                data-onfido-qa="documentExampleLabelGlare">
+                {translate('image_quality_guide.no_glare')}
+              </div>
+            </div>
+            <div className={style.documentExampleCol}>
+              <div
+                role="img"
+                aria-label={imgAltText}
+                className={classNames(
+                  style.documentExampleImg,
+                  style.documentExampleImgGood
+                )}
+              />
+              <div
+                className={style.documentExampleLabel}
+                data-onfido-qa="documentExampleLabelGood">
+                {translate('image_quality_guide.all_good')}
+              </div>
+            </div>
+          </div>
+          {isDesktop ? (
+            <CustomFileInput
+              className={classNames(
+                style.desktopUpload,
+                style.passportUploadContainer
+              )}
+              onChange={this.handleFileSelected}
+            >
+              <UploadButton text={buttonText} />
+            </CustomFileInput>
+          ) : (
+            <CustomFileInput
+              className={style.buttonContainer}
+              onChange={this.handleFileSelected}
+              accept="image/*"
+              capture
+            >
+              <UploadButton text={buttonText} />
+            </CustomFileInput>
+          )}
+          {error && <UploadError {...{ error, translate }} />}
```

Comment:
Yes. But I was a bit unsure about whether it should remain above the CTA or below. Visually, it will still be below a primary button. On other document type upload it is in between a primary button ("Continue on phone" button) and the upload link button.

Reply:
Attached screenshots as comparison for discussion: 
<img width="350" alt="Screenshot 2020-06-08 at 17 53 40" src="https://user-images.githubusercontent.com/2497081/84058382-12ec4100-a9b1-11ea-868e-6d535d8b7807.png">
<img width="350" alt="Screenshot 2020-06-08 at 14 47 45" src="https://user-images.githubusercontent.com/2497081/84058570-5777dc80-a9b1-11ea-9311-4ca7d35274d9.png">

Reply:
Interesting. I thought we wanted the CTA to always seat at the bottom of each screen, but if we decide to have errors below the CTA that would change things slightly. Would we leave a space for the error or would the button move up when the error appears? @raemarielee any thoughts?

Reply:
I agree – I think the CTA should have a consistent position at the bottom. Having the error directly above the button seems more ideal – especially since it's an important bit of info that the user should read first.

Reply:
OK. I'll make the change. :)